DSCResources/cChocoPackageInstall/cChocoPackageInstall.psm1

```diff
@@ -266,19 +262,16 @@ function UninstallPackage
     if (-not ($pParams))
     {
         Write-Verbose -Message 'Uninstalling Package Standard'
-        $packageUninstallOuput = choco uninstall $pName -y
+        $packageUninstallOuput = Invoke-Chocolatey "uninstall $pName -y"
     }
     elseif ($pParams)
     {
         Write-Verbose -Message "Uninstalling Package with params $pParams"
-        $packageUninstallOuput = choco uninstall $pName --params="$pParams" -y
+        $packageUninstallOuput = Invoke-Chocolatey "uninstall $pName --params=`"$pParams`" -y"
     }
 
     Write-Verbose -Message "Package uninstall output $packageUninstallOuput "
 
-    # Clear Package Cache
-    Get-ChocoInstalledPackage 'Purge'

```

Comment:
Same here?

Reply:
he purge command can cause errors and hangs code in DSC configurations.

Reply:
See previous comment.

Reply:
I added the purge command again.